bioblend/galaxy/objects/client.py

```diff
@@ -245,6 +244,10 @@ def import_new(self, src):
           dump of the workflow (this is normally obtained by exporting
           a workflow from Galaxy).
 
+        :type publish: bool
+        :param publish:  if ``True`` the uploaded workflow will be sharable;
```

Comment:
Can you substitute `sharable` with `published` here and below, please?

Reply:
Fixed by https://github.com/galaxyproject/bioblend/pull/247/commits/ecfddd5e09e75b2a69468324bbde42f4c5025031